lib/Optimize.cpp

```diff
@@ -235,7 +235,8 @@ void OptimizeModule(const EntityLifter &lifter, llvm::Module &module) {
   // makes code easier to read and analyze. This is a fairly narrow optimization
   // but it comes up often enough for lifted code.
 
-  AddConvertIntegerToPointerOperations(fpm);
+  // NOTE(alex): Need to rewrite this pass to somehow not rely on typed pointers.
+  // AddConvertIntegerToPointerOperations(fpm);
```

Comment:
This looks like a decent chunk of work. I'm thinking of trying to get this change in first and following up with a change to rewrite this pass.

Does this work? Or is this pass really integral to the working of Anvill.

Reply:
635b8200d902d168325ee81e298ac2c200fef0f7